packages/protocol/scripts/snaefellsjokull-poap-list.js

```diff
@@ -0,0 +1,124 @@
+"use strict";
+const ethers = require("ethers");
+const fs = require("fs");
+
+const L2_RPC_ENDPOINT = "http://l2rpc.a1.taiko.xyz";
+const L2_BLOCK_START_HEIGHT = 0;
+// 512862 timestamp: Thu, 05 Jan 2023 23:59:52 GMT
+// 512863 timestamp: Fri, 06 Jan 2023 00:00:16 GMT
+const L2_BLOCK_END_HEIGHT = 512862;
+
+const GOLDEN_TOUCH_ADDRESS = "0x0000777735367b36bC9B61C50022d9D0700dB4Ec";
+const L2_BRIDGE_ADDRESS = "0x0000777700000000000000000000000000000004";
+
+// Any user with a bridging transaction + one other type of transaction (transfer, dapp tx)
+const REASON_USED_BRIDGE = "REASON_USED_BRIDGE";
+const REASON_USED_OTHER_TYPES_OF_TXS = "REASON_USED_OTHER_TYPES_OF_TXS";
+const BATCH_SIZE = 5000;
+
+class Account {
+  constructor(address, reason) {
+    this.address = address;
+    this.reasons = [reason];
+  }
+}
+
+class AccountsList {
+  constructor() {
+    this.accountList = [];
+  }
+
+  addAccountWithReason(address, reason) {
+    const idx = this.accountList.findIndex(
+      (account) => account.address === address
+    );
+
+    if (idx < 0) {
+      this.accountList.push(new Account(address, reason));
+      return;
+    }
+
+    if (!this.accountList[idx].reasons.includes(reason)) {
+      this.accountList[idx].reasons.push(reason);
+    }
+  }
+
+  getEligibleAccounts() {
+    return this.accountList.filter((account) => account.reasons.length >= 2);
```

Comment:
Lets also sort the addresses by the number of txs.

Reply:
Added